common/lib/xmodule/xmodule/tests/test_sequence.py

```diff
@@ -45,7 +43,7 @@ def _set_up_course_xml(cls):
         """
         Sets up and returns XML course structure.
         """
-        course = xml.CourseFactory.build()
+        course = xml.CourseFactory.build(end=str(TOMORROW))
```

Comment:
Are there any potential flakiness issues with using `TOMORROW` here?  For example, if this test runs at 23:59:59?